hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/CapacityVolumeChoosingPolicy.java

```diff
@@ -46,9 +46,8 @@ public class CapacityVolumeChoosingPolicy implements VolumeChoosingPolicy {
 
   // Stores the index of the next volume to be returned.
   private final Random random = new Random();
-
   @Override
-  public HddsVolume chooseVolume(List<HddsVolume> volumes,
+  public synchronized HddsVolume chooseVolume(List<HddsVolume> volumes,
```

Comment:
@peterxcli , can you remove this synchronized word, also for RoundRobinVolumeChoosingPolicy.java?